internal/sourcecode/scan.go

```diff
@@ -601,6 +602,42 @@ For detailed documentation, see https://fly.dev/docs/django/
 	return s, nil
 }
 
+// setup Laravel with a sqlite database
+func configureLaravel(sourceDir string) (*SourceInfo, error) {
+	// Laravel projects contain the `artisan` command, all php projects contain `composer.json` (in theory!)
+	if !checksPass(sourceDir, fileExists("artisan", "composer.json")) {
+		return nil, nil
+	}
+
```

Comment:
If I understand correctly, you want to find *both* of these files to be pretty confident we're detecting Laravel. 

Under its current behaviour, `fileExists()` will return `true` if it finds *any*. (https://github.com/superfly/flyctl/issues/1001)

Reply:
I remembered that, yep! Artisan is the most important by far, but I figured `composer.json` was a good approximation for something PHP-ish if not. Honestly this could just be `artisan` if we wanted.

Reply:
Let's just use `artisan` then for now.